src/Illuminate/Auth/Passwords/DatabaseTokenRepository.php

```diff
@@ -10,73 +10,17 @@
 
 class DatabaseTokenRepository implements TokenRepositoryInterface
 {
-    /**
-     * The database connection instance.
-     *
-     * @var \Illuminate\Database\ConnectionInterface
-     */
-    protected $connection;
-
-    /**
-     * The Hasher implementation.
-     *
-     * @var \Illuminate\Contracts\Hashing\Hasher
-     */
-    protected $hasher;
-
-    /**
-     * The token database table.
-     *
-     * @var string
-     */
-    protected $table;
-
-    /**
-     * The hashing key.
-     *
-     * @var string
-     */
-    protected $hashKey;
-
-    /**
-     * The number of seconds a token should last.
-     *
-     * @var int
-     */
-    protected $expires;
-
-    /**
-     * Minimum number of seconds before re-redefining the token.
-     *
-     * @var int
-     */
-    protected $throttle;
-
     /**
      * Create a new token repository instance.
-     *
-     * @param  \Illuminate\Database\ConnectionInterface  $connection
-     * @param  \Illuminate\Contracts\Hashing\Hasher  $hasher
-     * @param  string  $table
-     * @param  string  $hashKey
-     * @param  int  $expires
-     * @param  int  $throttle
-     * @return void
      */
     public function __construct(
-        ConnectionInterface $connection,
-        HasherContract $hasher,
-        $table,
-        $hashKey,
-        $expires = 60,
-        $throttle = 60,
+        protected ConnectionInterface $connection,
+        protected HasherContract $hasher,
+        protected string $table,
+        protected string $hashKey,
+        protected int $expires = 3600,
```

Comment:
Wouldn't it make sense to rename the property to `$expires(In)Seconds` to make this more clear? Before you say, that someone relies on named properties: Laravel warns not to do that.

Also, the method could also accept `CarbonInterval` and `DateInterval`.

Reply:
I'm not making more changes than I have to in this PR.

Reply:
Sure, why not doing one PR per line 🤷🏻